### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#13] [General]
# Discord

Check if Discord is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/394e5756-0044-407a-8843-73d129987872

**Comment by OpenSource03** [comment]
More info on https://support.discord.com/hc/en-us/community/posts/360054793351-Discord-for-Windows-10-ARM64

**Comment by Mitia99** [comment]
It's worth mentioning [ArmCord](https://github.com/ArmCord/ArmCord) as an alternative native solution in this thread.

**Reply by viveksh1** [reply]
Yes, now called [LegCord ](https://github.com/Legcord/Legcord)
